stores/moderationSubmissionsStore.ts

```diff
@@ -2,18 +2,25 @@ import { gql } from 'graphql-request'
 import { defineStore } from 'pinia'
 import { ref, type Ref } from 'vue'
 import { gqlClient } from '../utils/graphql.js'
-import { type Submission } from '~/typedefs/gqlTypes.js'
+import type { Maybe, HealthcareProfessionalSubmission, Submission } from '~/typedefs/gqlTypes.js'
 
 export enum SelectedSubmissionListViewTab {
     ForReview = 'FOR_REVIEW',
     Approved = 'APPROVED',
     Rejected = 'REJECTED'
 }
 
+export enum SelectedSubmissionType {
+    Facilities = 'FACILITIES',
+    HealthcareProfessionals = 'HEALTHCARE_PROFESSIONALS'
+}
+
 export const useModerationSubmissionsStore = defineStore(
     'submissionsStore',
     () => {
         const submissionsData: Ref<Submission[]> = ref([])
+        const submissionHealtchareProfessionalsData: Ref< (Maybe<HealthcareProfessionalSubmission[]> | undefined)[]>
+        = ref([])
```

Comment:
We want to fetch the healthcare professionals instead of the healthcare professional submissions. Same for the Facilities